HeliPort/AppDelegate.swift

```diff
@@ -36,8 +36,19 @@ class AppDelegate: NSObject, NSApplicationDelegate {
 
         _ = ioctl_get(Int32(IOCTL_80211_DRIVER_INFO.rawValue), &drv_info, MemoryLayout<ioctl_driver_info>.size)
 
-        let version = String(cString: &drv_info.driver_version.0)
-        let interface = String(cString: &drv_info.bsd_name.0)
+        
+        let version = withUnsafePointer(to: &drv_info.driver_version) {
+            $0.withMemoryRebound(to: UInt8.self, capacity: MemoryLayout.size(ofValue: $0)) {
+                String(cString: $0)
+            }
+        }
```

Comment:
I assume this is for fixing compatibility with newer Swift compilers, please convert this into a string extension. 

I've wrote a similar fix a while back, not sure if it still works, might be still worth as an example.
```swift
public extension String {
    init<T>(cCharArray: T) {
        self = withUnsafeBytes(of: cCharArray) {
            $0.withMemoryRebound(to: CChar.self) { String(cString: $0.baseAddress!)}
        }
    }
}
```

Reply:
I will have a look.

Reply:
Implemented the extension, thank you.